--- origsrc/gnet-2.0.8/src/Makefile.am	2008-02-03 06:04:40.000000000 -0600
+++ src/gnet-2.0.8/src/Makefile.am	2008-02-04 06:50:55.843750000 -0600
@@ -1,7 +1,8 @@
 ## Process this file with automake to produce Makefile.in
 
-if OS_WIN32
 no_undefined = -no-undefined
+
+if OS_WIN32
 lws2_32 = -lws2_32
 endif
 
@@ -16,7 +17,9 @@
 
 libgnet_2_0_la_LDFLAGS = \
 	$(no_undefined) \
-	-version-info $(LT_CURRENT):$(LT_REVISION):$(LT_AGE) \
+	-version-info $(LT_CURRENT):$(LT_REVISION):$(LT_AGE)
+
+libgnet_2_0_la_LIBADD = \
 	$(GLIB_LIBS) $(GTHREAD_LIBS) $(lws2_32)
 
 libgnet_2_0_la_SOURCES = 	\
